community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/ast/convert/commands/ExpressionConverters.scala

```diff
@@ -90,7 +87,7 @@ object ExpressionConverters {
       case e: ast.GetDegree => e.asCommandGetDegree
       case e: ast.PrefixSeekRangeWrapper => e.asCommandStringSeekRange
       case e: ast.InequalitySeekRangeWrapper => e.asCommandValueSeekRange
-      case e: ast.AndedPropertyInequalities => e.asCommandAnds
+      case e: ast.AndedPropertyInequalities => e.asCommandAndedPropertyComparablePredicates
```

Comment:
So the pattern here is to call stuff `asCommand<nameOfTypeToConvert>`, which indicates that the type to convert will become a command. This name is only used in this match and where it is declared (line 486), so minor implications for a long name.

But perhaps you are objecting with the name `AndedPropertyComparablePredicates`? I agree it is very long and a bit heavy, do you have a better name in mind? Basically it works as a grouping type for comparable predicates where one side is a property, with an implicit logical AND between each.